src/image-tags.js

```diff
@@ -112,6 +113,12 @@ function pickImage({ pageType, config, seoConfig, data, url }) {
  */
 export function ImageTags(seoConfig, config, pageType, data, { url = {} }) {
   const { image, alt, includesHost = false } = pickImage({ pageType, data, url, seoConfig, config });
+  const story = get(data, ["data", "story"]);
+  const publisherConfig = get(data, ["config", "publisher-attributes"], {});
+  const fallbackValue = story ? false : true;
```

Comment:
Can we think of a better logic to handle this? This line of code seems wrongs to me.

Reply:
ok